atom/browser/native_window.cc

```diff
@@ -97,6 +99,7 @@ NativeWindow::NativeWindow(content::WebContents* web_contents,
       zoom_factor_(1.0),
       weak_factory_(this) {
   printing::PrintViewManagerBasic::CreateForWebContents(web_contents);
+  printing::PrintPreviewMessageHandler::CreateForWebContents(web_contents);
```

Comment:
Having looked into more I found it is actually necessary. Can move this call into `CommonWebContentsDelegate::InitWithWebContents` so we don't have to call it in two places?